tasks/2017/intro-to-zulip-mobile.md

```diff
@@ -0,0 +1,59 @@
+# GCI Tasks: Intro to Zulip mobile development (React Native)
+
+## Prerequisites
+
+* A working Zulip mobile development environment. See
+  https://github.com/zulip/zulip-mobile/blob/master/docs/developer-guide.md#dev-environment for instructions on how to set one up.
+
+* Update your working copy of [zulip-mobile](https://github.com/zulip/zulip-mobile) and then create a feature branch. [Learn
+  how](../../before-every-task.md).
+
+### Practice Running zulip-mobile
+
+Run app using
+```
+react-native run-ios // on Mac OS, to run on iOS device (simulator or device)
+react-native run-android // to run on Android device (emulator or device)
+```
+
+If you are running on Android emulator, then make sure it is launched before you
+run command on terminal.
+
+Enter server url (we recommend to use https://chat.zulip.org) and your login credentials.
+Open left drawer and narrow to `test here` stream and send a message with your
+name as the topic. And share your experience with us in `mobile` stream with
+topic as `feedback: setting up environment | <your name>` (replace `<your name>`
+with your name).
+
+Now task is to
+* share your learning with us on https://chat.zulip.org/#narrow/stream/mobile
+* updating [docs](https://github.com/zulip/zulip-mobile/tree/master/docs)
+* take screenshot of the app.
+
+**Tips:**
+After running app, answering following questions will help you in getting ideas
```

Comment:
After running the app, answering the following questions will help you write your docs: